src/server/stream_family.cc

```diff
@@ -1319,6 +1323,97 @@ OpResult<uint32_t> OpAck(const OpArgs& op_args, string_view key, string_view gna
   return acknowledged;
 }
 
+OpResult<ClaimInfo> OpAutoClaim(const OpArgs& op_args, string_view key, const ClaimOpts& opts) {
+  OpResult<pair<stream*, streamCG*>> cgr_res = FindGroup(op_args, key, opts.group);
+  if (!cgr_res)
+    return cgr_res.status();
+  stream* stream = cgr_res->first;
+  streamCG* group = cgr_res->second;
+
+  if (stream == nullptr || group == nullptr) {
+    return OpStatus::KEY_NOTFOUND;
+  }
+
+  streamConsumer* consumer = nullptr;
+  int64_t attempts = opts.count * 10;
```

Comment:
Add a comment that this logic is copied from Redis?
Reading this hard-coded 10 felt odd, so I looked at the docs to verify